### PR TITLE
Add community guidelines entity 

### PR DESCRIPTION
**Pull request by techsmyth**
New entity to hold community guidelines, which at the moment is very simple: a single profile.

### CODE REVIEW

File: src/domain/collaboration/collaboration/collaboration.interface.ts [hunk 1]
```diff
@@ -8,10 +8,6 @@ import { IInnovationFlow } from '../innovation-flow/innovation.flow.interface';
 
 @ObjectType('Collaboration')
 export abstract class ICollaboration extends IAuthorizable {
-  @Field(() => [ICallout], {
-    nullable: true,
-    description: 'List of callouts',
-  })
```
Comment (alstojanovic) [inline src/domain/collaboration/collaboration/collaboration.interface.ts:14]: Why is this removed?

Comment (techsmyth) [reply src/domain/collaboration/collaboration/collaboration.interface.ts:14]: There is a dedicated field resolver, so this is a bug

**Review by valentinyanakiev — APPROVED** [review]
